site/layouts/partials/home/masthead.html

```diff
@@ -6,7 +6,7 @@
           <span class="fw-semibold">
```

Comment:
In the actual prod version, we don't have the bold "New!":

![Screenshot 2024-07-30 at 07 15 46](https://github.com/user-attachments/assets/199984a6-a8e1-4896-bd06-d81b5e5c220f)

I don't have the context and history of these modifications, but if we want to achieve the exact same result as the one in production, we should get rid of this `<span>`.